lib/matplotlib/offsetbox.py

```diff
@@ -1507,28 +1507,21 @@ def contains(self, mouseevent):
         inside, info = self._default_contains(mouseevent)
         if inside is not None:
             return inside, info
-
-        xy_pixel = self._get_position_xy(None)
-        if not self._check_xy(None, xy_pixel):
+        if not self._check_xy(self._get_position_xy(None)):
```

Comment:
Isn't `_check_xy` supposed to take a renderer? This is passing a pixel position.

Reply:
ugh, sorry...